config/esp32/components/chip/Kconfig

```diff
@@ -150,6 +150,14 @@ menu "CHIP Core"
             help
                 Option to enable/disable CHIP log level filtering APIs.
 
+        config CHIP_SYSTEM_CONFIG_POOL_USE_HEAP
+            bool "Use heap memory for system buffer pool"
+            default n
+            help
+                If enabled, all the allocations done using
+                chip::Platform::MemoryAlloc() will allocate memory from ESP32's heap area.
+                Device having the PSRAM can enable this.
```

Comment:
That is .... not what CHIP_SYSTEM_CONFIG_POOL_USE_HEAP does, though.

Reply:
@bzbarsky-apple I have modified the help content, please take a look.